Amazon.QLDB.Driver/driver/QldbDriver.cs

```diff
@@ -325,18 +321,13 @@ internal T Execute<T>(Func<TransactionExecutor, T> func, RetryPolicy retryPolicy
 
         internal QldbSession GetSession()
         {
-            this.driverBase.LogSessionPoolState();
-
-            if (this.driverBase.PoolPermits.Wait(QldbDriverBase<QldbSession>.DefaultTimeoutInMs))
+            QldbSession session = this.driverBase.GetSessionFromPool();
```

Comment:
Can this be replaced with something like:
`return this.driverBase.GetSessionFromPool() ?? this.StartNewSession();`
?

Reply:
Done.